### PR TITLE
Widgets: Add Social Media Icons Widget

### LINKED ISSUE #2019: Widgets: add in the Social Icons widget from WordPress.com
See https://en.blog.wordpress.com/2015/04/17/social-media-icons-widget/

### PR DESCRIPTION
**Pull request by dereksmart**
Initial commit to Jetpack, Ported over from wpcom.

(plus an additional commit to another file to add it to our module description, which is not synced)

Fixes #2019
Will replace PR #2129

### CODE REVIEW

File: modules/widgets/social-media-icons.php [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+		$html = $args['before_widget'];
+
+		if ( ! empty( $instance['title'] ) ) {
+			$html .= $args['before_title'] . esc_html( $instance['title'] ) . $args['after_title'];
+		}
+
+		// display output
+		$html .= '<ul>';
+
+		$alt_text = esc_attr__( 'View %1$s&#8217;s profile on %2$s' );
```
Comment (jeherve) [inline modules/widgets/social-media-icons.php]: I think we miss the `jetpack` textdomain here.

File: modules/widgets/social-media-icons.php [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+			'title'              => __( 'Social', 'jetpack' ),
+			'facebook_username'  => '',
+			'twitter_username'   => '',
+			'instagram_username' => '',
+			'pinterest_username' => '',
+			'linkedin_username'  => '',
+			'github_username'    => '',
+			'youtube_username'   => '',
+			'vimeo_username'     => '',
+		);
```
Comment (roccotripaldi) [inline modules/widgets/social-media-icons.php]: No support for WordPress.com ?

Comment (dereksmart) [inline modules/widgets/social-media-icons.php]: not this time around :)

**Comment by roccotripaldi** [comment]
LGTM! working as expected. Icons look shiny.

**Comment by neeltjechanterie** [comment]
Add behance icon please 😃

**Comment by jeherve** [comment]
@neeltjechanterie I created a new issue (#7758) to keep track of your request.

**Pull request merged**